### ISSUE TITLE
Simplifying print(msg, file=sys.stderr) + sys.exit(1) to sys.exit(msg)

### ISSUE THREAD

**Issue by ulfalizer** [PyCQA/pylint#3094]
Don't know how messy this diagnostic would be to implement or what the potential for false positives is, so just throwing it out there:

Stuff like

```python
print(<msg>, file=sys.stderr)
sys.exit(1)
```

can be simplified to

```python
sys.exit(<msg>)
```

Might be nice to have a diagnostic for it.

"Consider replacing write to stderr followed by sys.exit(1) with sys.exit(string)" maybe.

**Comment by PCManticore** [comment]
Not sure how common this idiom actually is, so right now I'm not entirely confident we'd want this pattern in. Could be a nice third party check though.
